AutomatedTesting/Gem/Code/Source/AutoGen/NetworkTestPlayerComponent.AutoComponent.xml

```diff
@@ -29,10 +29,10 @@
         <Param Type="float" Name="SomeFloat" />
     </RemoteProcedure>
 
-    <RemoteProcedure Name="AuthorityToClientNoParams" InvokeFrom="Authority" HandleOn="Client" IsPublic="false" IsReliable="false" GenerateEventBindings="true" Description="" />
+    <RemoteProcedure Name="AuthorityToClientNoParams_PlaySomeSuperficialFx" InvokeFrom="Authority" HandleOn="Client" IsPublic="false" IsReliable="true" GenerateEventBindings="true" Description="" />
```

Comment:
Nit: _PlaySomeSuperficialFx, the superficial in here throws me off, maybe just _PlaySampleFx? or _PlayExampleFx?